rclpy/CMakeLists.txt

```diff
@@ -129,9 +129,12 @@ endfunction()
 
 add_library(rclpy_common SHARED
   src/rclpy_common/src/common.c
+  src/rclpy_common/src/handle.c
+  # List C++ sources last to ensure pybind11 pragmas
+  # involving the Py_DEBUG macro override those from
+  # CPython
```

Comment:
Oy, this is fragile.  Will this particular issue go away once rclpy is "pure" pybind11?

Reply:
Yeah, this issue is only a consequence of using both C and C++ files in the `rclpy_common` library. Once it's all C++ then we can rely on pybind11.h to fix the linking pragmas for us.